dashboard/app/controllers/pd/regional_partner_contact_controller.rb

```diff
@@ -1,18 +1,27 @@
-class Pd::RegionalPartnerContactController < ApplicationController
-  load_resource :regional_partner_contact, class: 'Pd::RegionalPartnerContact', id_param: :contact_id, only: [:thanks]
+require 'honeybadger/ruby'
+
+# Note: as of February 2020, this controller is deprecated
+# in favor of the regional_partner_mini_contact form.
+# That form is embedded at educate/professional-learning/contact-regional-partner,
+# which is what we redirect to here.
+# Not fully deprecating yet in case regional partners or Code.org staff
```

Comment:
Language nit: You are fully _deprecating_ this controller (that is, there's no reason to add anything new here).  You are not fully _removing_ it yet in case of saved links.

Reply:
Understood, will update!